content/episodes.json

```diff
@@ -1,5 +1,122 @@
 [
   {
+    "episode": "004",
+    "title":"ES6 and a Six Pack",
+    "description":"ECMAScript 6 was a major release for the JavaScript language. There’s a lot of great new features that have been added. In this episode we discuss how you can start using ES6 today and what we’ll start seeing in the next year for the latest update to ECMAScript. We share our favorite features and the best ways to learn and stay up to date with new features.",
+    "id": "",
+    "published": "April 04, 2016",
+    "panel": [
+      "Derrick Showers",
+      "Jem Young",
+      "Ryan Anklam",
+      "Ryan Burgess",
+      "Brian Holt",
+      "Augustus Yuan",
+      "Sarah Federman"
+    ],
+    "guests": [
+    ],
+    "links": [
```

Comment:
If you can add them here that'd be great! I just merged to master but feel free to add them.